### DISCUSSION TITLE
add_OnChange가 정의되지 않았다는 오류가 뜨고있습니다

### DISCUSSION THREAD

**Discussion by gwililgwi** [naverz/zepeto-studio-kor#570] [5. Q&A]
ClientStarter.js:47: TypeError: Cannot read property 'add_OnChange' of undefined

TypeError: Cannot read property 'add_OnChange' of undefined

멀티플레이 영상을 보고 따라 쳤으나 시작 시 저 오류가 뜨면서
제 캐릭터와 함께 베이스 캐릭터가 보여집니다.

어디서 틀린건지 감이 안잡혀 여기에 올려요

**Comment by pumfit** [comment]
안녕하세요! 문제가 되는 부분의 영상 위치를 알려주시면 좀 더 이유를 알기 쉬울 것 같습니다. 몇편의 어느 구간일까요?

**Reply by gwililgwi** [reply]
안녕하세요!! 감사합니다!
구간보다는 1,2,3 의 세팅을 끝낸 후 4편에서 똑같이 플레이를 눌렀을때 위와 같은 오류가 발생하고 있습니다.
2번까지 플레이했을 때 오류는 없었습니다.

**Reply by gwililgwi** [reply]
> 안녕하세요! 문제가 되는 부분의 영상 위치를 알려주시면 좀 더 이유를 알기 쉬울 것 같습니다. 몇편의 어느 구간일까요?

혹시 제가 controller라고 해서 만들었었던 스크립트 때문일까요??
지금보니 동일하게 start가 있고 이 코드에 영향을 받는것 같습니다

**Reply by gwililgwi** [reply]
> 안녕하세요! 문제가 되는 부분의 영상 위치를 알려주시면 좀 더 이유를 알기 쉬울 것 같습니다. 몇편의 어느 구간일까요?

아 서버 배우기 전 테스트 당시 만들었던 컨트롤러 스크립트 때문에 생성되었던걸 확인했습니다.
해결했습니다!!!
답변 달아주셔서 감사합니다!
